src/scripts/wildBattle/Routes.ts

```diff
@@ -1528,60 +1528,705 @@ GALAR
+    }),
+    [new RouteKillRequirement(10, GameConstants.Region.galar, 9)]
+));
+Routes.add(new RegionRoute(
+    'Galar Route 2 Lakeside', GameConstants.Region.galar, 11,
+    new RoutePokemon({
+        land: ['Lotad', 'Seedot', 'Purrloin', 'Blipbug', 'Gossifleur','Chewtle', 'Zigzagoon (Galar)', 'Nickit', 'Yamper', 'Obstagoon'],
+        water: ['Lapras', 'Drednaw', 'Gyarados', 'Arrokuda','Barraskewda', 'Magikarp', 'Feebas'],
+    }),
+    [new GymBadgeRequirement(BadgeEnums.Galar_Rock)]
+));
+Routes.add(new RegionRoute(
+    'Galar Route 9', GameConstants.Region.galar, 12,
+    new RoutePokemon({
+        land: ['Cramorant', 'Octillery', 'Kingler', 'Pyukumuku', 'Gastrodon (east)', 'Jellicent', 'Mareanie', 'Glalie'],
+        water: ['Wishiwashi', 'Qwilfish', 'Mantyke'],
+    }),
+    [new GymBadgeRequirement(BadgeEnums.Galar_Rock)]
+));
+Routes.add(new RegionRoute(
+    'Circhester Bay', GameConstants.Region.galar, 13,
+    new RoutePokemon({
+        land: ['Gastrodon (east)', 'Inkay', 'Cramorant', 'Octillery', 'Barbaracle', 'Bergmite', 'Toxapex', 'Dhelmise', 'Clobbopus'],
+        water: ['Lapras', 'Mantine', 'Mantyke', 'Wailmer', 'Wailord', 'Remoraid', 'Grapploct'],
+        headbutt: ['Greedent'],
+    }),
+    [new RouteKillRequirement(10, GameConstants.Region.galar, 12)]
+));
+Routes.add(new RegionRoute(
+    'Outer Spikemuth', GameConstants.Region.galar, 14,
+    new RoutePokemon({
+        land: ['Liepard', 'Thievul', 'Perrserker','Morpeko', 'Morpeko (hangry)', 'Jellicent', 'Bergmite', 'Mareanie','Toxapex', 'Dhelmise', 'Clobbopus'],
+        headbutt: ['Greedent'],
+    }),
+    [new RouteKillRequirement(10, GameConstants.Region.galar, 13)]
+));
+Routes.add(new RegionRoute(
+    'Galar Route 10 Station', GameConstants.Region.galar, 15,
+    new RoutePokemon({
+        land: ['Snom', 'Rhydon', 'Darumaka (Galar)', 'Vanillish', 'Klang','Mr. Mime (Galar)', 'Glalie', 'Snover', 'Vanilluxe', 'Cubchoo'],
+    }),
+    [new GymBadgeRequirement(BadgeEnums.Galar_Dragon)]
+));
+Routes.add(new RegionRoute(
+    'Galar Route 10 North', GameConstants.Region.galar, 16,
+    new RoutePokemon({
+        land: ['Snom', 'Duraludon', 'Stonjourner', 'Eiscue', 'Eiscue (noice)', 'Beartic', 'Vanillish', 'Abomasnow', 'Darumaka (Galar)', 'Mr. Mime (Galar)', 'Sneasel', 'Snover', 'Cubchoo'],
+    }),
+    [new RouteKillRequirement(10, GameConstants.Region.galar, 15)]
+));
+
+/*
+ISLE OF ARMOR
+*/
+Routes.add(new RegionRoute(
+    'Fields of Honor', GameConstants.Region.armor, 1,
+    new RoutePokemon({
+        land: ['Slowpoke (Galar)', 'Buneary', 'Jigglypuff', 'Abra', 'Klefki', 'Blipbug', 'Happiny'],
+        water: ['Magikarp', 'Remoraid', 'Octillery', 'Tentacool', 'Wingull', 'Mantyke', 'Starmie'],
+        special:
+        [
+            new SpecialRoutePokemon('Chansey', 'Kingler', 'Rockruff', 'Lopunny', 'Toxapex', 'Comfey' [new WeatherRequirement([WeatherType.Clear])]),
```

Comment:
A lot of the `SpecialRoutePokemon` are missing a `,` before the requirements array.
```suggestion
            new SpecialRoutePokemon('Chansey', 'Kingler', 'Rockruff', 'Lopunny', 'Toxapex', 'Comfey', [new WeatherRequirement([WeatherType.Clear])]),
```

Reply:
So... Would that fix the problem, I was under the belief that I had to rewrite the whole thing be Pokémon instead of be weather?

Reply:
Not currently, they all need to be separated into their own line.
But I could change it for you guys so that it instead accepts an array of Pokémon so you just need to add `[]` around the Pokémon names?

Reply:
If that could be done in reasonable time it would be awesome! It would.... Save literal days on mine and Krush's part, but if it would be a lot of work to change then that would defeat the purpose of saving time.

Reply:
Sweet, gimme like 10 minutes

Reply:
👍 Should be good to go now!
So you just need 1 row per weather type.

Reply:
how did you do that in like 5 minutes?

Reply:
Thanks, I have no idea how you did that so quickly, but it saves me from near insanity, so... Cheers!